rules/xcodeproj.bzl

```diff
@@ -640,12 +641,34 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
             "run": scheme_action_details,
         }
 
+        scheme_infos = _get_attr_values_for_name(ctx.attr.additional_scheme_infos, AdditionalSchemeInfo, "additional_scheme_info")
```

Comment:
If you go with the suggestion above, this would be:
`scheme_infos = [target[AdditionalSchemeInfo] for target in ctx.attr.additional_scheme_infos]`